src/main/kotlin/model/Lotto.kt

```diff
@@ -1,12 +1,11 @@
 package model
 
-class Lotto(val purchaseAmount: Int) {
+class Lotto(val purchaseAmount: Int, val manualTicketAmount: Int, val manualTickets: Tickets) {
     val numberOfTickets: Int = purchaseAmount / PURCHASE_AMOUNT_UNIT
-    lateinit var tickets: Tickets
+    var tickets = manualTickets
 
     fun generateTickets() {
```

Comment:
```suggestion
    fun generateTickets(manualTicketAmount: Int) {
```

You can pass `manualTicketAmount` as a method parameter rather than use a class property.

Reply:
Yes, did it. Should we always turn a rarely used class property into a method parameter to keep the code clean?

Reply:
Well, not always.
But in this case, I think passing the parameter makes sense, since it feels more natural for `Lotto` to just have the manual tickets, rather than having both the number of manual tickets and the manual tickets.